app/assets/stylesheets/darkswarm/_shop-inputs.scss

```diff
@@ -6,93 +6,110 @@
 .darkswarm {
   // #search
   @include placeholder(rgba(0, 0, 0, 0.4), #777);
+}
 
-  // ordering
-  product {
-    input {
-      @include border-radius(0);
-
-      @include csstrans;
-
-      margin: 0;
-      width: 10rem;
-      display: inline;
-
-      @include box-shadow(none);
-
-      border-color: #b3b3b3;
-      text-align: right;
-
-      @include breakpoint(desktop) {
-        width: 8rem;
-      }
-
-      @include breakpoint(tablet) {
-        width: 7rem;
-      }
-
-      @include breakpoint(phablet) {
-        float: left !important;
-        font-size: 0.75rem;
-        padding-left: 0.25rem;
-        padding-right: 0.25rem;
-      }
-
-      @include breakpoint(mobile) {
-        width: 5.8rem;
-      }
-
-      &:hover {
-        @include box-shadow(none);
+.reveal-modal.product-bulk-modal {
+  width: 26em;
+}
 
-        border-color: #888;
-        background-color: #fafafa;
-      }
+// Components to add variants to cart and change quantities
+//
+// They are not nested so that they can be used in modals.
+button.add-variant, button.variant-quantity {
+  height: 2.5rem;
+  border-radius: 0;
+  background-color: $orange-500;
+  color: white;
+  // Override foundation button styles:
+  font-size: 1rem;
+  margin: 0;
+  padding: 0;
+  transition: none;
+
+  &:hover {
+    background-color: $orange-600;
+  }
+  &[disabled] {
```

Comment:
there are some issues in code climate we could fix like here:
"Rule declaration should be followed by an empty line"

Reply:
This happened with previous mobile PRs as well, `codeclimate` doesn't run on non-master branches...